README.md

```diff
@@ -113,7 +121,7 @@ Let's go through an example. Given the following template, `helloWorld.html`, we
 </template>
 ```
 
-We know out of the box the `lightning-button` will be handled by the package automatically. `foo-fancy-button`, however, will need to be resolved. First, create a `jest.config.js` file at the root of the SFDX project workspace and add the following:
+We know out of the box the `lightning-button` will be handled by the package automatically. `foo-fancy-button`, however, will need to be resolved. First, create a `jest.config.js` file at the root of the Salesforce DX project workspace and add the following:
```

Comment:
Because it's in the `lightning` namespace, the `lightning-button` just works. However, you must write some code to help the Jest resolver find the `foo-fancy-button` component.